### PR TITLE
First pass at adding ood coding standards.

### PR DESCRIPTION
**Pull request by Oglopf**
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1135148780858012/1202699232514765) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by Oglopf** [comment]
I tried to DRY this out the best I could from what is already there, but other eyes please double check me. I tried to make it "short" but it still feels a bit too long, if anyone sees something superfluous or confusing please feel free to rip it out.

**Review by johrstrom — DISMISSED** [review]
LGTM though you may have already broken you branch naming style 😆!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -55,6 +83,260 @@ In addition to the [RuboCop] styles configured described above, we follow common
...
+    end
+
+    def conditions_true?
+      if var1 && bool2 && x > y || big % small > 1
+    end
+    ```
+
+### Classe and Modules
+
+* Camel case for Class and Module names.
```
Comment (gerald-byrket) [inline CONTRIBUTING.md:338]: Just to make sure we are clear.  Some people view camel case as 
class camelCase  
some view camel case
class CamelCase

Which is your intent here?
I do understand that the 2nd example is Mountain Case.  However, we should probably make it very clear.

**Review by gerald-byrket — CHANGES_REQUESTED** [review]

**Review by gerald-byrket — CHANGES_REQUESTED** [review]
whoops

**Comment by Oglopf** [comment]
@johrstrom Technically speaking, without the formalism in place I could have named the branch `beer_mugs_and_ping_pong` and it would be valid.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Oglopf) [reply CONTRIBUTING.md]: Thanks for that! I should have had Mountain/Pascal there. Changes made with example added.

**Review by gerald-byrket — APPROVED** [review]
LGTM.  Thanks for the hard work on this.

**Pull request merged**